trucks.js

```diff
@@ -504,8 +506,75 @@ var foodTrucks = [
 ];
 
 // this module should support the following methods:
-// getTrucks() - return all trucks
-// getTruck(name) - return the truck object matching 'name'
-// getFoodTypes() - return unique list of all associated food types (underscore has a function to help)
-// filterByDay(day) - return trucks with 'day' in schedule (use your filterByDay function from Module 3 homework)
-// filterByFoodType(foodType) - return trucks with associated 'foodType'
+
+var truckCalls = {
+
+	// getTrucks() - return all trucks
+	getTrucks: function() {
+
+		var allTrucks = [];
+		
+		for (truck in foodTrucks) {
+			allTrucks.push(foodTrucks[truck]['name'])
```

Comment:
This step is unnecessary. You already have an array, foodTrucks, that you can just return directly.